server/applicationset/applicationset_helpers.go

```diff
@@ -0,0 +1,211 @@
+package applicationset
```

Comment:
IMO moving several methods into a separate file not necessary making code more readable. On the other hand it will make it more difficult to cherry-pick fixes and create unnecessary maintenance work